.gitlab-ci.yml

```diff
@@ -21,7 +21,8 @@ workflow:
     - if: $CI_COMMIT_BRANCH
 
 variables:
-  CI_IMAGE: !reference [.ci-unified, variables, CI_IMAGE]
+  # CI_IMAGE: !reference [.ci-unified, variables, CI_IMAGE]
```

Comment:
is this needed?

Reply:
yes, I'll revert it back when we change the image in the common settings